tests/integration/test_subscriptions.py

```
@@ -0,0 +1,890 @@
+
+    def test_get_resource_data_returns_none_when_empty(self) -> None:
+        mgr = SubscriptionManager()
+        assert mgr.get_resource_data("nonexistent") is None
+
+    def test_get_resource_data_returns_stored_data(self) -> None:
+        from unraid_mcp.core.types import SubscriptionData
+
+        mgr = SubscriptionManager()
+        mgr.resource_data["test"] = SubscriptionData(
+            data={"key": "value"},
+            last_updated=datetime.now(),
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)